setup.py

```diff
@@ -25,6 +28,12 @@
             requires=REQUIRES,
             scripts=SCRIPTS)
 
+extensions = [Extension('scilpy.tractanalysis.robust_streamlines_metrics',
```

Comment:
Please add in the readme the command to build scilpy: `python setup.py build_ext --inplace`

Reply:
Done, but not tested

Reply:
Works fine in a new virtualenv